### PR TITLE
[Refactor] Update results and ckpts

### PR DESCRIPTION
**Pull request by liqikai9**
## Motivation

Update verified HRNet + COCO models and their checkpoints and logs with 1.x.

Update HRNet + PoseTrack18 test results on detector bboxes.

## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: configs/body_2d_keypoint/topdown_heatmap/posetrack18/hrnet_posetrack18.md [hunk 1]
```diff
@@ -34,22 +34,22 @@
...
+| [pose_hrnet_w32](/configs/body_2d_keypoint/topdown_heatmap/posetrack18/td-hm_hrnet-w32_8xb64-20e_posetrack18-384x288.py) |  384x288   | 87.1 | 89.0 | 85.1  | 80.2 | 80.6 | 82.8 | 79.6 | 83.7  | [ckpt](https://download.openmmlab.com/mmpose/top_down/hrnet/hrnet_w32_posetrack18_384x288-806f00a3_20211130.pth) | [log](https://download.openmmlab.com/mmpose/top_down/hrnet/hrnet_w32_posetrack18_384x288_20211130.log.json) |
+| [pose_hrnet_w48](/configs/body_2d_keypoint/topdown_heatmap/posetrack18/td-hm_hrnet-w48_8xb64-20e_posetrack18-256x192.py) |  256x192   | 88.3 | 90.2 | 86 .0 | 81.0 | 80.7 | 83.3 | 80.6 | 84.6  | [ckpt](https://download.openmmlab.com/mmpose/top_down/hrnet/hrnet_w48_posetrack18_256x192-b5d9b3f1_20211130.pth) | [log](https://download.openmmlab.com/mmpose/top_down/hrnet/hrnet_w48_posetrack18_256x192_20211130.log.json) |
```
Comment (Ben-Louis) [inline configs/body_2d_keypoint/topdown_heatmap/posetrack18/hrnet_posetrack18.md:41]: '86 .0' -> '86.0'

Comment (liqikai9) [reply configs/body_2d_keypoint/topdown_heatmap/posetrack18/hrnet_posetrack18.md]: Fixed.

**Review by ly015 — APPROVED** [review]

**Comment by ly015** [comment]
@liqikai9 Please resolve the conflits.

**Comment by ly015** [comment]
Lint seems faild.

**Review by ly015 — APPROVED** [review]

**Pull request merged**